target/linux/ramips/mt7621/base-files/etc/board.d/02_network

```diff
@@ -82,6 +82,11 @@ ramips_setup_macs()
 		wan_mac=$(mtd_get_mac_ascii u-boot-env et1macaddr)
 		label_mac=$(mtd_get_mac_binary factory 0x4)
 		;;
+	cudy,wr2100)
+		lan_mac=$(mtd_get_mac_binary bdinfo 0xde00) # "factory 0x4" also does it
```

Comment:
This is already set in DTS with gmac0 and can be removed.

Reply:
done